src/plone/distribution/core.py

```diff
@@ -10,7 +10,8 @@
 import json
 
 
-DEFAULT_SCHEMA = json.load(open(DEFAULT_PATH / "schema.json"))
+with (DEFAULT_PATH / "schema.json").open() as fp:
+    DEFAULT_SCHEMA = json.load(fp)
```

Comment:
You could also do:

```suggestion
DEFAULT_SCHEMA = json.load((DEFAULT_PATH / "schema.json").read_bytes())
```

Reply:
bytes or text? 🤔 but yes, I love this `Path.read_(bytes|text)()` methods ⭐

Reply:
@gforcada `json.loads` accepts either, but I picked bytes to avoid the possibility of decoding errors on systems that don't use utf-8 as the default encoding.